client/app/components/ParameterValueInput.less

```
@@ -0,0 +1,19 @@
+@import '~antd/lib/input-number/style/index'; // for ant @vars
+
+.parameter-apply-button {
+  margin-left: -55px;
+  width: 50px;
+}
+
+.parameter-input {
+  width: 195px !important;
+
+  &--apply-button {
+    padding-right: 60px !important;
```

To make it equal the 4px from the right edge.

```suggestion
    padding-right: 58px !important;
```